apps/medon-fe/src/components/AvailabilityCalendar/useCalendar.tsx

```diff
@@ -0,0 +1,152 @@
+import { useCallback, useState } from 'react';
+import { Event } from 'react-big-calendar';
+import { useTranslation } from 'react-i18next';
+import moment from 'moment';
+import { toast } from 'react-toastify';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { useForm } from 'react-hook-form';
+
+import 'react-big-calendar/lib/css/react-big-calendar.css';
+import { hoursSchema } from 'validation/selectHourRangeSchema';
+import { toastConfig } from 'utils/toastConfig';
+
+interface SelectHours {
+  start: number;
+  end: number;
+}
+
+export function useCalendar() {
```

Comment:
i believe in terms of hooks, or you put in the hook file that Oleksi created or rename this file to hooks.tsx and use it here, from what i remember of the structure

Reply:
renamed